server/chart_config.json

```diff
@@ -1242,6 +1304,31 @@
       "unit": "%"
     }
   },
+  {
+    "category": "Demographics",
```

Comment:
If there is a way to plot Y-axis log-scale, it might be useful here

Reply:
Good point, that would be a larger change here. I can add a feature requirement.